Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+node {
+    try {
+        stage('Pre-build') {
+            checkout scm
+            sh 'git submodule update --init --recursive'
+        }
+
+        stage 'Build'
+        def image = docker.build "fsae-2020-${BUILD_TAG.toLowerCase()}"
+
+//        stage('Lint') {
```

Comment:
>single line comments
gross, but I'll allow it.

Reply:
http://vimdoc.sourceforge.net/htmldoc/visual.html#blockwise-operators